cmd/skaffold/app/cmd/config/util.go

```diff
@@ -94,12 +94,18 @@ func GetConfigForKubectx() (*ContextConfig, error) {
 	if global {
 		return cfg.Global, nil
 	}
+
+	localContext := kubecontext
```

Comment:
TAG_TOO_LATE Too late, `resolveKubectlContext` could already have set the `global` flag (condition cmd/skaffold/app/cmd/config/util.go:45).